source/common/common/logger.h

```diff
@@ -72,6 +72,8 @@ class Logger {
     off = spdlog::level::off
   } levels;
 
+  static std::string default_log_format;
```

Comment:
const please, and probably `DEFAULT_LOG_FORMAT`

Reply:
`static const char* DEFAULT_LOG_FORMAT` now.